src/App.jsx

```diff
@@ -1,8 +1,50 @@
 import './App.css'
+import Button from "./components/Button.jsx";
```

Comment:
Je hebt netjes de Button, Product en Tile geimporteerd maar de onderstaande niet:
import bag1 from './assets/bag_1.png'
import bag2 from './assets/bag_2.png'
import bag3 from './assets/bag_3.png'
import bag4 from './assets/bag_4.png'
import brand from './assets/brand.png'
import our_story from './assets/our_story.png'

Hier is het de bedoeling om de tassen te inporteren zodat je later in je code alleen de naam tussen de accoaldes hoeft te zetten. ZIe mijn comment regel 28.

voor de rest is het allemaal netjes.